### PR TITLE
docs(ZITADEL): uppercase name & OAuth 2.0 / OIDC 1.0 (#1)

### PR DESCRIPTION
**Pull request by fforootd**
#### Preliminary checks

* [x] I have [read the Code of Conduct](https://github.com/kdeldycke/awesome-iam/blob/main/.github/code-of-conduct.md)
* [x] I applied all rules from the [Contributing guide](https://github.com/kdeldycke/awesome-iam/blob/main/.github/contributing.md)
* [x] I have checked there is not other [Issues](https://github.com/kdeldycke/awesome-iam/issues) or [Pull Requests](https://github.com/kdeldycke/awesome-iam/pulls) covering the same topic to open

#### Summary

We changed the wording slightly and also mentioned that ZITADEL serves OIDC and OAuth.

Kudos for this GH Repo it's an impressive collection! 
If you are interested, we would really like to chat with you.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -318,6 +318,8 @@ The old *OpenID* is dead; the new *OpenID Connect* is very much not-dead.
 
 - [authentik](https://goauthentik.io/?#correctness) - Open-source Identity Provider similar to Keycloak.
 
+- [ZITADEL](https://github.com/caos/zitadel/) - ZITADEL is an Open-Source, Cloud-Native Identity and Access Management with support for OpenID Connect 1.0 and OAuth 2.0 to easily integrate your projects.
```
Comment (fforootd) [inline readme.md:321]:
```suggestion
- [ZITADEL](https://github.com/caos/zitadel) - ZITADEL is an Open-Source, Cloud-Native Identity and Access Management with support for OpenID Connect 1.0 and OAuth 2.0 to easily integrate your projects.
```

**Comment by fforootd** [comment]
Hi there

I did quickly check the [linters error message](https://github.com/kdeldycke/awesome-iam/pull/63/checks?check_run_id=2676766154#step:3:11) and it's true that it is the same url. 

As ZITADEL can do account management and oidc, the tricky question is wether you want to have a somewhat redundant entry or not.

**Comment by kdeldycke** [comment]
Thanks @fforootd for the contribution! I'm ready to merge it but unfortunately duplicate links are not allowed by `awesome` guidelines. You'll have to choose between the `OAuth2 & OpenID` or `Account Management` section. Both are fine to me.

**Comment by fforootd** [comment]
Ok thanks for the feedback.

So i will merge both texts and move them to the `OAuth2 & OpenID` section

**Comment by fforootd** [comment]
@kdeldycke i did merge the sections, hopefully the text is not to long now 😁

**Comment by kdeldycke** [comment]
No that's fine. Thanks @fforootd for the quick fix. Let's merge!

**Comment by fforootd** [comment]
Thank you!

**Pull request merged**